pandapower/create.py

```diff
@@ -1970,6 +1973,9 @@ def create_lines(net, from_buses, to_buses, length_km, std_type, name=None, inde
             create_line(net, "line1", from_bus=0, to_bus=1, length_km=0.1, std_type="NAYY 4x50 SE")
 
     """
+    from_buses = [from_buses] if type(from_buses) == int else from_buses
```

Comment:
what is the reason for this?

Reply:
User-friedlyness: even if the user only wants to create one line (as one is also a multiple of lines) and only passes the index as an int instead of a list, the function will work. 

Also this could be a way to reduce the number of functions the user has to know by possibly replacing create_line(). 

What do you think about it?